src/Status/Update.elm

```diff
@@ -0,0 +1,18 @@
+module Status.Update exposing (update)
+
+import Types exposing (Msg)
+import Status.Types exposing (StatusModel, StatusMsg(GetStatus, NewStatus), StatusResponse)
+import Status.Api exposing (getStatus)
+
+
+update : StatusMsg -> StatusModel -> ( StatusModel, Cmd Msg )
+update msg model =
+    case msg of
+        GetStatus ->
+            ( model, getStatus )
+
+        NewStatus (Ok response) ->
+            ( { model | response = Just response }, Cmd.none )
+
+        NewStatus (Err err) ->
+            ( model, Cmd.none )
```

Comment:
subtle (potential) bug -- a failure in the api request after a success will never be shown to the user. It's unlikely, but if someone goes to the status page, jumps back to the silences page, and then returns to status at some later time, this page will only update on a successful response.

not too important because it's the status page, but a nice thing that's handled if we use the `ApiData` thing.